.github/workflows/rnmobile-ios-runner.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
         strategy:
             matrix:
-                xcode: ['13.3.1']
+                xcode: ['14.2']
```

Comment:
For the `macos-12` runner the latest Xcode 14 version is `14.2` ([docs](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#xcode))